### PR TITLE
@mczernek/update google and crashlytics

### PR DESCRIPTION
**Pull request by mczernek**
# Why

Some users reports they are warned by apple for using 'UIWebView'. While we don't use it in our code, those libraries in previous version seemed to be doing that.

I've tested FaceDetector and Ad Mob , and it works.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -7,3 +7,4 @@ ios/Pods/GoogleMobileVision/Detector/Frameworks/GoogleMobileVision.framework/Goo
 ios/Pods/GoogleMobileVision/FaceDetector/Frameworks/FaceDetector.framework/FaceDetector filter=lfs diff=lfs merge=lfs -text
```
Comment (sjchmiela) [inline .gitattributes]: Are we able to remove this line now?

File: ios/Exponent.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3143,7 +3145,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = host.exp.Exponent;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore host.exp.Exponent";
+				PROVISIONING_PROFILE_SPECIFIER = "App Store for MCzernek";
```
Comment (sjchmiela) [inline ios/Exponent.xcodeproj/project.pbxproj]: I think some of those changes should be reverted 🤔

File: ios/Exponent/ExponentDebug.entitlements [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
```
Comment (sjchmiela) [inline ios/Exponent/ExponentDebug.entitlements]: This too probably shouldn't land on `master`?

**Review by sjchmiela — APPROVED** [review]
Apart from these couple of comments LGTM 👍 

Let's also see what CI will say

File: .gitattributes (hunk 1 above)
Comment (mczernek) [reply .gitattributes]: Yes, we can!

**Review by ide — APPROVED** [review]

**Pull request merged**